### PR TITLE
Fix the GitHub CI build

### PR DESCRIPTION
**Pull request by mrobinson**
This PR fixes various issues that were preventing the CI from successfully running.

### CODE REVIEW

**Comment by mrobinson** [comment]
Okay. After a lot of wrestling with cargo and GitHub runners, I think this PR is ready to be merged. It gets the CI working again.

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -44,9 +67,8 @@ jobs:
         gst-inspect-1.0 | grep Total
     - name: Build
       run: cargo build
-    - name: Examples
-      run: |
-        ls examples/*.rs | xargs -I{} basename  {} .rs  | grep -v params_connect | RUST_BACKTRACE=1 GST_DEBUG=3 xargs -I{} cargo ex {} --all-features
```
Comment (Manishearth) [inline .github/workflows/rust.yml:49]: Why aren't we running the examples anymore?

Comment (Manishearth) [reply .github/workflows/rust.yml:49]: feel free to r=me to fix the immediate problem though

Comment (mrobinson) [reply .github/workflows/rust.yml:49]: At least for me, one of these examples did not terminate on my local machine. Perhaps I didn't wait long enough for that to happen though. I'll play around a bit locally.

Comment (mrobinson) [reply .github/workflows/rust.yml]: It seems this works on CI while running the examples, so I have removed that comit.

**Comment by mrobinson** [comment]
Thanks for the review!

**Pull request merged**